spec/integ/matrix-client-crypto.spec.js

```diff
@@ -689,4 +689,62 @@ describe("MatrixClient crypto", function() {
                 return aliTestClient.httpBackend.flush('/keys/query', 1);
             });
     });
+
+    it("React on device_one_time_keys send by /sync", function() {
```

Comment:
The description here should explain what we are expecting to happen. "uploads an appropriate number of one-time keys given the /sync response" or something.